### PR TITLE
Add insecureTLS config for queue-worker to chart

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add a config value to enable insecure tls on the queue-worker.

## Motivation and Context

The queue-worker supports enabling insecure tls using an env variable. Make this configuration available in the OpenFaaS Chart.

## How Has This Been Tested?

1. Install OpenFaaS Pro with the updated chart and `--set queueWorkerPro.insecureTLS=true`
```bash
helm upgrade openfaas --install chart/openfaas \                    
    --namespace openfaas  \
    --set functionNamespace=openfaas-fn \
    --set generateBasicAuth=true \
    --set openfaasPro=true \
    --set autoscaler.enabled=true \
    --set queueWorkerPro.insecureTLS=true
```

2. Verify the deploy was successful and the `tls_insecure` env variable set correctly.
```
kubectl describe deploy/queue-worker -n openfaas | grep tls_insecure
       tls_insecure:           true
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
Can you make this only work for when openfaasPro is set to true?

**Comment by welteki** [comment]
> Can you make this only work for when openfaasPro is set to true?

Should it then only be an option for queueWorkerPro, `--set queueWorkerPro.insecureTLS=true` or do we keep it under the queueWorker options?

File: chart/openfaas/values.yaml [hunk 1]
```diff
@@ -204,6 +204,7 @@ queueWorker:
   gatewayInvoke: true
```
Comment (alexellis) [inline chart/openfaas/values.yaml:204]: I think that we can remove `gatewayInvoke` now, also in the template

File: chart/openfaas/values.yaml [hunk 2]
```diff
@@ -204,6 +204,7 @@ queueWorker:
...
   queueGroup: "faas"
   ackWait: "60s"
+  insecureTLS: false
```
Comment (alexellis) [inline chart/openfaas/values.yaml:207]: I think it'd work under the queue-worker Pro section

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**